### PR TITLE
Sensu Enterprise ServiceNow integration

### PR DESCRIPTION
**Pull request by portertech**

### CODE REVIEW

File: source/docs/0.23/enterprise-built-in-handlers.md [hunk 1]
```diff
@@ -447,6 +448,142 @@ timeout
     "timeout": 30
     ~~~
 
+### ServiceNow
+
+Create [ServiceNow][servicenow] configuration items upon [Sensu client registration][client-registration], and create/resolve [ServiceNow][servicenow] incidents for [Sensu events][events].
```
Comment (calebhailey) [inline source/docs/0.23/enterprise-built-in-handlers.md]: maybe drop the second servicenow.com link in the same sentence?

Comment (calebhailey) [inline source/docs/0.23/enterprise-built-in-handlers.md]: Also, if we can link to a servicenow reference doc for configuration items that would be :+1:

Comment (calebhailey) [inline source/docs/0.23/enterprise-built-in-handlers.md]: ...maybe just link to the CMDB marketing stuff from this paragraph: http://www.servicenow.com/products/it-service-automation-applications/configuration-management.html

File: source/docs/0.23/enterprise-built-in-handlers.md [hunk 2]
```diff
@@ -447,6 +448,142 @@ timeout
...
+    "user": "admin",
+    "password": "secret",
+    "create_cmdb_ci": true,
+    "cmdb_ci_table": "cmdb_ci_server",
+    "timeout": 10
+  }
+}
+~~~
+
+When creating ServiceNow configuration items, by default, Sensu will use the client's name for the item name. Individual Sensu clients can override the name of their corresponding configuration item, using specific client definition attributes. In addition to specifying a item name, any valid CMDB table attributes (fields & values) may also be set, e.g. `os_version`.
```
Comment (calebhailey) [inline source/docs/0.23/enterprise-built-in-handlers.md]: Some useful links for "configuration items" in this context: 
- http://wiki.servicenow.com/index.php?title=CMDB_Data_Model#gsc.tab=0 
- http://wiki.servicenow.com/index.php?title=Introduction_to_Assets_and_Configuration#gsc.tab=0

**Comment by agoddard** [comment]
🍧 :shipit:

**Pull request merged**